model/src/main/scala/za/co/absa/atum/model/utils/JsonImplicits.scala

```diff
@@ -16,16 +16,23 @@
 
 package za.co.absa.atum.model.utils
 
-object OptionImplicits {
-  implicit class OptionEnhancements[T](val option: Option[T]) extends AnyVal {
-    /**
-      * Gets the `option` value or throws the provided exception
-      *
-      * @param exception the exception to throw in case the `option` is None
-      * @return
-      */
-    def getOrThrow(exception: => Throwable): T = {
-      option.getOrElse(throw exception)
+import io.circe.{Encoder, Decoder}
+import io.circe.syntax._
+import io.circe.parser.decode
+
+object JsonUtils {
+  def asJson[T: Encoder](obj: T): String = {
+    obj.asJson.noSpaces
+  }
+
+  def asJsonPretty[T: Encoder](obj: T): String = {
+    obj.asJson.spaces2
+  }
+
+  def fromJson[T: Decoder](jsonStr: String): T = {
+    decode[T](jsonStr) match {
+      case Right(value) => value
+      case Left(error) => throw new RuntimeException(s"Failed to decode JSON: $error")
```

Comment:
**For discussion:**
I would rename these methods to
`asJsonString`, `asJsonStringPretty` and `fromJsonString`
To make ti clear it accepts `String`, instead of _Circe_'s `Json` and actually not to confuse with functions of the same name in _Circe_.

Reply:
Yah, I can change them. I don't see anything wrong with that.

Reply:
@benedeki good suggestion, it's better to have them named differently than methods in Circe.